src/main/index.js

```diff
@@ -1043,7 +1054,21 @@ function runApp() {
     mainWindow.webContents.send('change-view', data)
   }
 
-  function setMenu() {
+  async function setMenu() {
+    const sidenavSettings = baseHandlers.settings._findSidenavSettings()
+    let hideTrendingVideos = false
+    try {
+      hideTrendingVideos = (await sidenavSettings.hideTrendingVideos).value
+    } catch {}
+    let hidePopularVideos = false
+    try {
+      hidePopularVideos = (await sidenavSettings.hidePopularVideos).value
+    } catch {}
+    let hidePlaylists = false
+    try {
+      hidePlaylists = (await sidenavSettings.hidePlaylists).value
+    } catch {}
```

Comment:
This can be simplified to this, as the error was occuring when it was trying to read the `value` property and `null` is falsey.
```suggestion
    const hideTrendingVideos = (await sidenavSettings.hideTrendingVideos)?.value
    const hidePopularVideos = (await sidenavSettings.hidePopularVideos)?.value
    const hidePlaylists = (await sidenavSettings.hidePlaylists)?.value
```

Reply:
I tested the change before suggesting it.